docs/DeveloperGuide.md

```diff
@@ -6,9 +6,28 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### AddSalesCommand
+#### Overview
+The AddSalesCommand is responsible for adding sales data to a Manga. The Sale data consists of three attributes,  
+`quantitySold`, `unitPrice` and `totalRevenue`. The `quantitySold` and `unitPrice` are inputs from the user, while
+`totalRevenue` is calculated by multiplying `quantitySold` and `unitPrice`.
 
 
+For the AddSalesCommand to be successful, the manga that the sales data is associated with must exist. If the `sales`
+command is successful, the `Sales` data is then saved via Storage.
+
+![mangasales_class.png](uml%2Fimages%2Fmangasales_class.png)
+#### Interaction
+
+The following sequence diagram illustrates the interactions that occur when the parser creates a new `AddSalesCommand`. 
+
+![addsalesdata.png](uml%2Fimages%2Faddsalesdata.png)
+
+The following object diagram illustrates object structure after the above interaction is successfully run
+with the input `sales -a Kubo Tite -m Bleach -q 10000 -p 11.90`.
+
+![mangasales_object.png](uml%2Fimages%2Fmangasales_object.png)
```

Comment:
<img width="377" alt="Screenshot 2024-10-30 at 12 54 34" src="https://github.com/user-attachments/assets/a8bdda4a-1940-4b26-857e-a2c7b48a5d03">
missing activation bars for Tantou and Parser